spec/features/manage_cmr/groups/filtering_groups_spec.rb

```diff
@@ -1,22 +1,50 @@
 # These tests are for the Groups Index page filtering by search boxes, which
 # filter by provider(s) and/or member(s)
-
 describe 'Filtering groups', reset_provider: true, js: true do
   before :all do
-    VCR.use_cassette('edl', record: :new_episodes) do
+    VCR.use_cassette("edl/#{File.basename(__FILE__, ".rb")}_vcr", record: :none) do
       create_group(
-        name: 'Group_1',
-        description: 'test group',
+        name: 'Group_3',
+        description: 'test group 3',
         provider_id: 'MMT_2',
         members: %w[qhw5mjoxgs2vjptmvzco]
       )
 
       create_group(
-        name: 'Group_2',
-        description: 'test group 2',
+        name: 'Group_4',
+        description: 'test group 4',
         provider_id: 'MMT_2',
         members: %w[qhw5mjoxgs2vjptmvzco rarxd5taqea q6ddmkhivmuhk]
       )
+
+      create_group(
+        name: 'Group_5',
+        description: 'test group 3',
```

Comment:
Minor change, description: 'test group 3', -> description: 'test group 5'. Just for consistency purposes

Reply:
solved